frontend/src/pages/Trip/components/TripBody/AboutTrip.js

```diff
@@ -0,0 +1,91 @@
+import React, { useState } from "react";
+import Modal from "../../../../modules/ui/Modal/Modal";
+
+function addDaysAndFormat(inputDate, numberOfDays) {
+    const date = new Date(inputDate);
+    date.setDate(date.getDate() + numberOfDays);
+
+    const months = [
+        "Jan", "Feb", "Mar", "Apr", "May", "Jun", "Jul", "Aug", "Sep", "Oct", "Nov", "Dec",
+    ];
```

Comment:
What is if introduce some common file for this, and import It here as this can be required at some other place as well, so we can use single file only.